tests/linear_test.py

```diff
@@ -0,0 +1,39 @@
+# pylint: disable=C,E1101,E1102
+import unittest
+
+import torch
+
+from e3nn.linear import Linear
+from e3nn import rs
+
+
+class Tests(unittest.TestCase):
+    def test1(self):
+        torch.set_default_dtype(torch.float64)
+        Rs_in = [(50, 0), (150, 1), (50, 0), (100, 2)]
+        Rs_out = [(2, 0), (1, 1), (1, 2), (3, 0)]
+
+        torch.manual_seed(0)
```

Comment:
This seems like an awkward test since it depends on a manual_seed producing a specific value.

I ran the following locally,
```
lin_vals = []
lin_mod_vals = []
for i in range(1000):
    Rs_in = [(5, 0), (3, 1), (2, 0), (7, 2)] 
    Rs_out = [(5, 0), (1, 1), (1, 2), (3, 0)] 
    torch.manual_seed(i)
    linmod = LinearMod(Rs_in, Rs_out)
    lin = Linear(Rs_in, Rs_out)
    N = 2
    input = torch.randn(1, N, rs.dim(Rs_in))
    lin_vals.append(lin(input).pow(2).mean(0).sub(1).abs().max().detach().numpy())
    lin_mod_vals.append(linmod(input).pow(2).mean(0).sub(1).abs().max().detach().numpy())
```
And got the following plots for Linear and my modified Linear code, respectively...
![image](https://user-images.githubusercontent.com/4080470/77874497-f047ea00-7201-11ea-8fe7-73a9fe03ee96.png)
![image](https://user-images.githubusercontent.com/4080470/77874507-fb027f00-7201-11ea-9edc-b9aa790d023c.png)

Is this what you would expect?

Reply:
What we want to test is that the output is gaussian.

```python
torch.set_default_dtype(torch.float64)
Rs_in = [(50, 0), (30, 1), (20, 0), (70, 2)] 
Rs_out = [(2, 0), (1, 1), (1, 2), (3, 0)]

lin = Linear(Rs_in, Rs_out)
features = torch.randn(10000, rs.dim(Rs_in))
features = lin(features)

plt.hist(features.detach().flatten(), density=True, bins=100)
x = torch.linspace(-3, 3, 100)
plt.plot(x, x.pow(2).div(-2).exp().div(math.sqrt(2 * math.pi)))
```
![image](https://user-images.githubusercontent.com/333780/77928241-7585da00-72a8-11ea-8246-a79c3da6ee9a.png)